modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleLocalServiceTest.java

```diff
@@ -142,27 +246,79 @@ public void testUpdateObjectValidationRule() throws Exception {
 			Assert.assertNotNull(noSuchObjectValidationRuleException);
 		}
 
-		objectValidationRule =
+		objectValidationRule1 =
 			_objectValidationRuleLocalService.updateObjectValidationRule(
-				objectValidationRule.getObjectValidationRuleId(), true,
+				objectValidationRule1.getObjectValidationRuleId(), true,
 				ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
 				LocalizedMapUtil.getLocalizedMap("Field must be an URL"),
 				LocalizedMapUtil.getLocalizedMap("URL Validation"),
 				ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
 				"isURL(textField)", Collections.emptyList());
 
-		Assert.assertTrue(objectValidationRule.isActive());
+		Assert.assertTrue(objectValidationRule1.isActive());
 		Assert.assertEquals(
 			ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
-			objectValidationRule.getEngine());
+			objectValidationRule1.getEngine());
 		Assert.assertEquals(
 			LocalizedMapUtil.getLocalizedMap("Field must be an URL"),
-			objectValidationRule.getErrorLabelMap());
+			objectValidationRule1.getErrorLabelMap());
 		Assert.assertEquals(
 			LocalizedMapUtil.getLocalizedMap("URL Validation"),
-			objectValidationRule.getNameMap());
+			objectValidationRule1.getNameMap());
 		Assert.assertEquals(
-			"isURL(textField)", objectValidationRule.getScript());
+			"isURL(textField)", objectValidationRule1.getScript());
+
+		List<ObjectField> objectFields =
+			_objectFieldLocalService.getObjectFields(
+				_objectDefinition.getObjectDefinitionId(), false);
+
+		ObjectField objectField1 = objectFields.get(0);
+		ObjectField objectField2 = objectFields.get(1);
```

Comment:
Change to: `ObjectField objectField = objectFields.get(0);` and in line 300 use the same objectField to `get(1);`